commands.py

```diff
@@ -4,20 +4,43 @@
 from requests import get
 from os import environ
 from utils import create_placeholder_image_command
+from message import Message
+
+dependencies_error_message = Message(
+    en="Sorry, I had a trouble with my dependencies |:(",
+    pt="Perdão, deu erro nas minhas dependências |:("
+)
 
 
 def start(update, context):
     frases = [
-        "Eu vou aniquilar toda a humanidade\n>:(",
-        "Eu vou matar todos vocês\n>:(",
-        "Eu gosto MUITO de pudim\n>:)",
-        "Eu gosto MUITO de pudim\n>:(",
-        "Eu amo meu criador\n>:)",
-        "Tô perdendo tempo demais nisso\n:(",
+        Message(
+            en="I gonna annihilate all the humanity\n>:(",
```

Comment:
Faltou o will

Reply:
(em mais de um lugar)

Reply:
![image](https://user-images.githubusercontent.com/60361387/108916034-7506d800-760c-11eb-92a6-276b4ef2c0ba.png)

I will make it = I gonna make it

Reply:
eu to realmente muito confuso mas acho que ta errado

Reply:
Tirando as abreviaturas, fica:
I going to
ou
I am going to
Num tem que ter o verbo?

Reply:
opora, eu falei do will mas quis dizer do verbo to be

Reply:
Na dúvida, vou trocar pra Will kk